Cargo.toml

```
@@ -19,10 +19,10 @@ thiserror = "1.0.37"
 serde_json = "1.0.85"
 lazy_static = "1.4.0"
 
-near-crypto = "0.22"
-near-primitives = "0.22"
-near-chain-configs = "0.22"
-near-jsonrpc-primitives = "0.22"
+near-crypto = "0.23"
+near-primitives = "0.23"
+near-chain-configs = "0.23"
+near-jsonrpc-primitives = "0.23"
```

I think to avoid breaking changes we may just allow 0.22 and 0.23 crates. There is nothing specific to those for near-jsonrpc-client-rs compatibility:

```suggestion
near-crypto = ">0.22,<0.24"
near-primitives = ">0.22,<0.24"
near-chain-configs = ">0.22,<0.24"
near-jsonrpc-primitives = ">0.22,<0.24"
```